2020/7xxx/CVE-2020-7377.json

```diff
@@ -1,18 +1,105 @@
 {
-    "data_type": "CVE",
-    "data_format": "MITRE",
-    "data_version": "4.0",
     "CVE_data_meta": {
+        "ASSIGNER": "cve@rapid7.com",
+        "DATE_PUBLIC": "2020-08-18T08:48:00.000Z",
         "ID": "CVE-2020-7377",
-        "ASSIGNER": "cve@mitre.org",
-        "STATE": "RESERVED"
+        "STATE": "PUBLIC",
+        "TITLE": "Rapid7 Metasploit Framework Pelative Path Traversal in telpho10_credential_dump module"
```

Comment:
"Pelative Path Traversal" - should be Relative Path Traversal?

Reply:
Fixed!